bigml/supervised.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+#
+# Copyright 2018 BigML
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+"""A local Predictive Supervised model class
+
+This module defines a supervised model to make predictions locally or
+embedded into your application without needing to send requests to
+BigML.io.
+
+This module cannot only save you a few credits, but also enormously
+reduce the latency for each prediction and let you use your logistic
+regressions offline.
+
+Example usage (assuming that you have previously set up the BIGML_USERNAME
+and BIGML_API_KEY environment variables and that you own the
+logisticregression/id below):
```

Comment:
the too paragraphs above mention logistic regression, but i think they mean supervised model?